dplace_app/api_views.py

```diff
@@ -221,21 +220,29 @@ def id_array(l):
             result_set.languages.add(lang)
 
     if 'c' in query_dict:
+        f = open('text.txt', 'w')
+        f.write(query_dict['c'][0].split('-')[0])
```

Comment:
Writing this text file was for debugging, right? So should be removed before putting the code into production.